src/app/routes/index.js

```diff
@@ -2,7 +2,16 @@ import article from './article';
 import cpsAsset from './cpsAsset';
 import home from './home';
 import radio from './radio';
+import onDemandRadio from './onDemandRadio';
 import error200 from './error200';
 import error404 from './error404';
 
-export default [article, cpsAsset, home, radio, error200, error404];
+export default [
+  article,
+  cpsAsset,
+  home,
+  radio,
```

Comment:
should maybe be renamed to `liveRadio` but as a separate PR, not this one.

Reply:
Done